coalib/results/Diff.py

```diff
@@ -257,4 +274,5 @@ def change_line(self, line_nr, original_line, replacement):
 
     def __eq__(self, other):
         return ((self._file == other._file) and
-                (self.modified == other.modified))
+                (self.modified == other.modified) and
+                (self.rename == other.rename))
```

Comment:
hey we have a generate_eq function in Decorators.py could you do another commit on top using that one?